README.md

```diff
@@ -32,7 +32,7 @@ Execute the following commands to build from the source:
  
 - To build the package:
    ```   
-   bal pack ./cosmosdb
+   bal build ./cosmosdb
```

Comment:
As we have changed the folder name, this needs to changed too 
```suggestion
   bal build ./ballerina
```
Same applies for `test` too.